### PR TITLE
fix: simple brightcove activity with fullscreen icon hidden

### PR DESCRIPTION
**Pull request by trevorah**
oct-24-2017 15-00-10

### CODE REVIEW

File: packages/brightcove-video/android/src/main/res/layout/brightcove_player_activity.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical" android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@android:color/black">
```
Comment (CoreFloDev) [inline packages/brightcove-video/android/src/main/res/layout/brightcove_player_activity.xml]: This elements shouldn't be needed, you should be able to set the background to black directly on the brightcoveExoPlayerView

File: packages/brightcove-video/android/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- This is used to style the buttons on the media controller -->
+    <style name="BrightcoveControlBar" parent="BrightcoveControlBarDefault">
+        <!-- You can override values as shown here. The commented variables have the default values
+             It's recommended to parent BrightcoveControlBarDefault -->
+        <item name="brightcove_full_screen">false</item>
+    </style>
```
Comment (CoreFloDev) [inline packages/brightcove-video/android/src/main/res/values/styles.xml]: Where did you find the documentation for that bit?

Comment (trevorah) [reply packages/brightcove-video/android/src/main/res/values/styles.xml]: https://github.com/BrightcoveOS/android-player-samples/tree/master/brightcove-player-controls

File: packages/brightcove-video/android/src/main/res/layout/brightcove_player_activity.xml (hunk 1 above)
Comment (trevorah) [reply packages/brightcove-video/android/src/main/res/layout/brightcove_player_activity.xml]: done!

**Pull request merged**